portafly/src/i18n/locales/en/audience/accounts/accounts.yml

```diff
@@ -0,0 +1,70 @@
+---
+#--------------------------Path: Accounts > Account overview ------------------------------------#
+# Filename subject to change----------------------------- ------------------------------------#
```

Comment:
We've been using `overview`, also the folder could be `account` (singular) instead.

Reply:
@josemigallas With regards to the filename change, we are adding more than just overview to the YAML file. Changing the name to overview.yml might be very limiting because account covers more than overview, but also the tab for account applications. What do you think?

Reply:
I see, well from the code perspective the "Overview page" includes both _overview_ and _applications_ tab but I see now that may be a bit confusing.
I have no strong opinions about this really, we could maybe use `accounts/account.yml`.